### ISSUE TITLE
Pre-work Checklist: Developer: Maria Herzer

### ISSUE THREAD

**Issue by mjh-projects** [hackforla/website#3911]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by mjh-projects** [comment]
I skipped the meeting and went to the other parts of this onboarding since I had time this eventing is that ok? 
So far I have spend a few  hours going over this installing github and trying to go through the pull request etc. I went through and did one as I was reading this to practice as I went along I hope this is ok.

**Comment by mjh-projects** [comment]
Some of the resource links as well as the visual studio github wiki have dead links. Wasn't sure what you do with that. I would assume that was covered in How to create issues(https://github.com/hackforla/website/wiki/How-to-create-issues) but that link was also down or perhaps it is already being worked on and I am unaware.

**Comment by jdingeman** [comment]
Hi @mjh-projects, I saw you had trouble with the links. Just wanted to confirm that you were able to read the links since the Wiki is back up?

**Comment by ExperimentsInHonesty** [comment]
@mjh-projects Did you see @jdingeman 's note above? https://github.com/hackforla/website/issues/3911#issuecomment-1428916188.

If you could keep us updated about your progress with your environment in this issue, that will come in handy later when we are reviewing the progress of all people onboarding.

**Comment by mjh-projects** [comment]
@ExperimentsInHonesty I did not but it seems the links are available now. 

I wasn't actually using visual studio at the time so just continued  with notepad++ and git but thought visual studio might be a better tool to use but found the dead links. I'll go through how to create an issue so that I know the reporting procedure for the future.

**Comment by mjh-projects** [comment]
@jdingeman  Thanks, all good with the links now.

**Comment by ExperimentsInHonesty** [comment]
@mjh-projects I am reopening this issue until you have been assigned to a development issue long enough to make an in progress update.  It can will stay open until you get an issue that takes longer than a week to complete (likely a medium or large size issue).  That way, we know you can make detailed progress updates.  In the meantime, if you get reminders to update this issue, just let us know what size and number issue you are assigned to at the time.  That way, we have a record (in one place) of your progress.

I see you finished your good first issue and second issue but have not yet picked up a small issue.  Here are the issues that are available that match the role labels you choose on your prework
[Front End, Small](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-7198257) | [Back End, Small](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257)

**Comment by ExperimentsInHonesty** [comment]
@ I see you finished your good first issue and second issue but have not yet picked up a small issue.  Here are the issues that are available that match the role labels you choose on your prework (there are currently 1 of each as of my sending you this link)
[Front End, Small](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-7198257) | [Back End, Small](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257)

**Comment by ExperimentsInHonesty** [comment]
@mjh-projects I see you finished your good first issue and second issue but have not yet picked up a small issue.  Here are the issues that are available that match the role labels you choose on your prework (there are currently 1 of each as of my sending you this link)
[Front End, Small](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+front+end%22#column-7198257) | [Back End, Small](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22complexity%3A+small%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257)

**Comment by mjh-projects** [comment]
@ExperimentsInHonesty  I will go ahead and check today. Thanks.

**Comment by jdingeman** [comment]
Hi @mjh-projects - please provide us with an update on your overall status. Thanks!

**Comment by mjh-projects** [comment]
Provide Update
1. Progress - I have gone through everything on this
2. Blockers - none 
3. Availability evenings After 6PM  except Wednesday
4. ETA - I think this is done?

**Comment by roslynwythe** [comment]
Hi @mjh-projects.  Congratulations on completing your `Complexity: Small` issue.  During periods when you are not working on any issues, we request that you post a comment in this issue to inform us of your progress, any problems you might be having, and to make sure that you are still interested in continuing with Hack for LA.  We would like to see updates, in this issue or in other issues, at least every two weeks.  

If we do not hear from you by 7/10/2023, we will assume that you are not interested in continuing with Hack for LA and we will close this issue, but you can reopen it at any time.  If you do reopen the issue, please move it into the "In Progress" column.  Thank you.

**Comment by roslynwythe** [comment]
Hi @mjh-projects as mentioned above in this comment https://github.com/hackforla/website/issues/3911#issuecomment-1618908889 we will be closing this issue, but you are welcome to reopen it in the future.

**Comment by HackforLABot** [comment]
## Activity Log: mjh-projects
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- mjh-projects opened: https://github.com/hackforla/website/issues/3911 at 01/31/2023, 08:58 PM PST
- mjh-projects assigned: https://github.com/hackforla/website/issues/3911 at 01/31/2023, 08:58 PM PST
- mjh-projects assigned: https://github.com/hackforla/website/issues/3893#issuecomment-1411478803 at 02/01/2023, 11:27 PM PST
- mjh-projects commented on: https://github.com/hackforla/website/issues/3893#issuecomment-1413272766 at 02/01/2023, 11:32 PM PST
- mjh-projects opened: https://github.com/hackforla/website/pull/3924 at 02/02/2023, 01:47 AM PST
- mjh-projects commented on: https://github.com/hackforla/website/issues/3911#issuecomment-1413458093 at 02/02/2023, 02:00 AM PST
- mjh-projects commented on: https://github.com/hackforla/website/issues/3911#issuecomment-1413517723 at 02/02/2023, 02:33 AM PST
- mjh-projects PR was merged: https://github.com/hackforla/website/pull/3924#event-8439314337 at 02/04/2023, 02:57 PM PST
- mjh-projects issue closed by PR 3924: https://github.com/hackforla/website/issues/3893#event-8439314361 at 02/04/2023, 02:57 PM PST
- mjh-projects commented on: https://github.com/hackforla/website/pull/3940#issuecomment-1421881393 at 02/07/2023, 06:17 PM PST
- mjh-projects submitted review: https://github.com/hackforla/website/pull/3940#pullrequestreview-1288439918 at 02/07/2023, 08:20 PM PST
- mjh-projects closed as completed: https://github.com/hackforla/website/issues/3115#event-8485622225 at 02/09/2023, 07:02 PM PST
- mjh-projects reopened: https://github.com/hackforla/website/issues/3115#event-8485622225 at 02/09/2023, 07:03 PM PST
- mjh-projects closed as completed: https://github.com/hackforla/website/issues/3911#event-8508606990 at 02/13/2023, 12:27 PM PST
- mjh-projects assigned: https://github.com/hackforla/website/issues/3950 at 02/13/2023, 12:34 PM PST
- mjh-projects commented on: https://github.com/hackforla/website/issues/3950#issuecomment-1428648244 at 02/13/2023, 12:38 PM PST
- mjh-projects commented on: https://github.com/hackforla/website/issues/3950#issuecomment-1436085969 at 02/19/2023, 12:30 PM PST
- mjh-projects commented on: https://github.com/hackforla/website/issues/3911#issuecomment-1445354624 at 02/26/2023, 04:55 AM PST
- mjh-projects commented on: https://github.com/hackforla/website/issues/3950#issuecomment-1445356749 at 02/26/2023, 05:03 AM PST
- mjh-projects commented on: https://github.com/hackforla/website/issues/3911#issuecomment-1445359370 at 02/26/2023, 05:14 AM PST
- mjh-projects closed as completed: https://github.com/hackforla/website/issues/3911#event-8609503758 at 02/26/2023, 05:14 AM PST
- mjh-projects issue closed by PR 4199: https://github.com/hackforla/website/issues/3950#event-8771860945 at 03/16/2023, 01:32 PM PDT
- mjh-projects commented on: https://github.com/hackforla/website/issues/3950#issuecomment-1473024302 at 03/16/2023, 07:30 PM PDT
- mjh-projects opened: https://github.com/hackforla/website/pull/4199 at 03/16/2023, 07:50 PM PDT
- mjh-projects PR was merged: https://github.com/hackforla/website/pull/4199#event-8786874986 at 03/18/2023, 09:02 PM PDT
- mjh-projects issue closed by PR 4199: https://github.com/hackforla/website/issues/3950#event-8786875039 at 03/18/2023, 09:02 PM PDT
- mjh-projects commented on: https://github.com/hackforla/website/pull/4208#issuecomment-1479992756 at 03/22/2023, 10:36 AM PDT
- mjh-projects submitted review: https://github.com/hackforla/website/pull/4208#pullrequestreview-1353157117 at 03/22/2023, 10:37 AM PDT
- mjh-projects commented on: https://github.com/hackforla/website/issues/3911#issuecomment-1485528349 at 03/27/2023, 10:11 AM PDT
- mjh-projects assigned: https://github.com/hackforla/website/issues/4211 at 03/27/2023, 10:16 AM PDT
- mjh-projects commented on: https://github.com/hackforla/website/issues/4211#issuecomment-1485534817 at 03/27/2023, 10:16 AM PDT
- mjh-projects opened: https://github.com/hackforla/website/pull/4340 at 03/29/2023, 02:48 AM PDT
- mjh-projects commented on: https://github.com/hackforla/website/pull/4309#issuecomment-1489056432 at 03/29/2023, 10:58 AM PDT
- mjh-projects commented on: https://github.com/hackforla/website/pull/4309#issuecomment-1489185352 at 03/29/2023, 12:31 PM PDT
- mjh-projects submitted review: https://github.com/hackforla/website/pull/4309#pullrequestreview-1363818930 at 03/29/2023, 12:35 PM PDT
- mjh-projects closed as completed: https://github.com/hackforla/website/issues/3229#event-8881565996 at 03/29/2023, 04:08 PM PDT
- mjh-projects reopened: https://github.com/hackforla/website/issues/3229#event-8881565996 at 03/29/2023, 04:26 PM PDT
- mjh-projects PR was merged: https://github.com/hackforla/website/pull/4340#event-8907819991 at 04/02/2023, 07:24 PM PDT
- mjh-projects issue closed by PR 4340: https://github.com/hackforla/website/issues/4211#event-8907820067 at 04/02/2023, 07:24 PM PDT
- mjh-projects commented on: https://github.com/hackforla/website/pull/4386#issuecomment-1495085332 at 04/03/2023, 03:49 PM PDT
- mjh-projects commented on: https://github.com/hackforla/website/pull/4386#issuecomment-1497784958 at 04/05/2023, 09:28 AM PDT
- mjh-projects submitted review: https://github.com/hackforla/website/pull/4386#pullrequestreview-1373290464 at 04/05/2023, 09:29 AM PDT
- mjh-projects commented on: https://github.com/hackforla/website/pull/4412#issuecomment-1498217649 at 04/05/2023, 03:06 PM PDT
- mjh-projects assigned: https://github.com/hackforla/website/issues/4050#issuecomment-1493490600 at 04/07/2023, 12:13 PM PDT
- mjh-projects commented on: https://github.com/hackforla/website/issues/4050#issuecomment-1500565112 at 04/07/2023, 12:14 PM PDT
- mjh-projects opened: https://github.com/hackforla/website/issues/4489 at 04/11/2023, 08:19 PM PDT
- mjh-projects commented on: https://github.com/hackforla/website/issues/4050#issuecomment-1513993276 at 04/18/2023, 06:10 PM PDT
- mjh-projects commented on: https://github.com/hackforla/website/issues/3911#issuecomment-1513995172 at 04/18/2023, 06:13 PM PDT
- mjh-projects commented on: https://github.com/hackforla/website/issues/4050#issuecomment-1522277064 at 04/25/2023, 12:04 PM PDT
- mjh-projects unassigned: https://github.com/hackforla/website/issues/4050#issuecomment-1547138979 at 05/14/2023, 08:24 PM PDT
- mjh-projects issue closed as completed: https://github.com/hackforla/website/issues/3911#event-9784028829 at 07/11/2023, 01:07 AM PDT
